### PR TITLE
[VMSS] add cmake

### PR DESCRIPTION
**Pull request by MVoz**

### CODE REVIEW

File: ports/mpir/portfile.cmake [hunk 1]
```diff
@@ -1,8 +1,4 @@
-include(vcpkg_common_functions)
-
-if(VCPKG_CMAKE_SYSTEM_NAME STREQUAL "WindowsStore")
-    message(FATAL_ERROR "MPIR currently can only be built for desktop")
-endif()
+ vcpkg_fail_port_install(MESSAGE "MPIR currently can only be built for desktop" ON_TARGET "UWP")
```
Comment (NancyLi1013) [inline ports/mpir/portfile.cmake:2]: Could you please update this
`configure_file(${SOURCE_PATH}/COPYING.lib ${CURRENT_PACKAGES_DIR}/share/mpir/copyright COPYONLY)`
as
`configure_file(${SOURCE_PATH}/COPYING.lib ${CURRENT_PACKAGES_DIR}/share/${PORT}/copyright COPYONLY)`?

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Review by NancyLi1013 — APPROVED** [review]

**Comment by Voskrese** [comment]
fix 
https://github.com/microsoft/STL/blob/master/azure-devops/provision-image.ps1

**Comment by Voskrese** [comment]
close it?

**Comment by BillyONeal** [comment]
@Voskrese Indeed, does https://github.com/microsoft/vcpkg/pull/10767 resolve this to your satisfaction?

**Comment by Voskrese** [comment]
@BillyONeal 
I have nothing to do with this, the main thing is that the service works and test the libraries

**Pull request closed without merging**